NeuroID/src/main/java/com/neuroid/tracker/utils/RootHelper.kt

```diff
@@ -117,7 +117,9 @@ class RootHelper {
 
     private fun isEmulatorFilesPresent(): Boolean {
         emulatorFiles.forEach { path ->
-            return path.exists()
+           if (path.exists()) {
+               return true
+           }
```

Comment:
because this is in a foreach, will it return from the function or just the forEach loop?

Reply:
some kotlin docs for ref: ```Kotlin has three structural jump expressions:
return by default returns from the nearest enclosing function or [anonymous function](https://kotlinlang.org/docs/lambdas.html#anonymous-functions).
break terminates the nearest enclosing loop.
continue proceeds to the next step of the nearest enclosing loop.```